pkg/api/routes.go

```diff
@@ -54,10 +54,10 @@ func (h *Handler) GetRoutes() []Route {
 			h.CreateImage,
 		},
 		{
-			"GetImageMapInfo",
+			"GetClientAccessInfo",
 			"GET",
-			"/image/mapinfo",
-			h.GetImageMapInfo,
+			"/client",
```

Comment:
Also, I liked the idea of something similar to `docker-machine env` that you can use in scripting to set everything to env vars. `eval "$(docker-machine env dev)"`.  That would be useful to then use s3cmd with minimal effort.
https://docs.docker.com/machine/reference/env/